ci/release/apply_wheel_modifications.sh

```diff
@@ -6,8 +6,12 @@
 VERSION=${1}
 CUDA_SUFFIX=${2}
 
+# pyproject.toml versions
 sed -i "s/^version = .*/version = \"${VERSION}\"/g" pyproject.toml
+
+# pyproject.toml cuda suffixes
 sed -i "s/^name = \"ucx-py\"/name = \"ucx-py${CUDA_SUFFIX}\"/g" pyproject.toml
+sed -i "s/cudf/cudf${CUDA_SUFFIX}/g" pyproject.toml
```

Comment:
I copied this approach from the way cudf installs rmm. https://github.com/rapidsai/cudf/blob/87a8ede8dcd9b6cd6e38c41f74daec316f48e7db/ci/release/apply_wheel_modifications.sh#L17